Replace string split by regex to avoid xamarin android issues (#1253)
microsoft/appcenter-sdk-dotnet: CHANGELOG.md

## Version 2.6.3 （Under development）

### AppCenter

#### Android

> * **[Fix]** Fix `MissingMethodException` that could occur with some build configurations (a bug introduced in version `2.5.0`).

Reviewer: Let's where the exception is thrown.

```suggestion
* **[Fix]** Fix `MissingMethodException` of `String.Split()` with some build configurations (a bug introduced in version `2.5.0`).
```
